src/features/Settings/Settings.tsx

```diff
@@ -10,20 +10,26 @@ import {
 } from '@services/Store';
 
 import { AccountList, Mobile, Desktop } from '@components';
-import { NetworkId, CustomNodeConfig } from '@types';
+import { NetworkId, CustomNodeConfig, StoreAsset } from '@types';
 import { DEFAULT_NETWORK, IS_ACTIVE_FEATURE } from '@config';
 import { BREAK_POINTS } from '@theme';
-import translate from '@translations';
+import translate, { translateRaw } from '@translations';
+import FlippablePanel from '@features/Settings/components/FlippablePanel';
+import { RatesContext } from '@services/RatesProvider';
+import { Balance, BalanceAccount } from '@features/Dashboard/components/WalletBreakdown/types';
```

Comment:
Perhaps we want to move these types somewhere more general then?

Reply:
dun